### DISCUSSION TITLE
Opinionated installation instructions for the less tech-savvy

### DISCUSSION THREAD

**Discussion by winstxnhdw** [winstxnhdw/lc-hax#294] [Show and tell]
Here's opinionated instructions for those who do not want a single thought to be generated from their brains.

First, install [Microsoft .NET 9.0 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/sdk-9.0.300-windows-x64-installer) and [Git](https://git-scm.com/download/win).

Press <kbd>Win</kbd> + <kbd>S</kbd> and search for `PowerShell` and open the program.

Copy and paste the following into `PowerShell`.

```bash
cd $env:USERPROFILE\Desktop
git clone --recursive --depth 50 https://github.com/winstxnhdw/lc-hax.git
cd lc-hax
explorer .
```

You can now build and install the dependencies by executing `requirements.bat`.

```bash
.\requirements.bat
```

Execute `launch.bat` when you have launched the game to inject the assembly into the game process.

```bash
.\launch.bat
```

**Comment by joep26020** [comment]
😂😂😂😂 people give up when they see it takes effort not. No exe doesn't mean it's hard to setup

**Reply by D1GQ** [reply]
It's a good way from keeping all the children from hacking on this game.

**Comment by heyzeuscrist** [comment]
how do i know if it worked

**Reply by winstxnhdw** [reply]
Go into the game and type a command in the chat.

**Reply by heyzeuscrist** [reply]
okay

**Reply by heyzeuscrist** [reply]
man it did not work

**Reply by winstxnhdw** [reply]
And you have your answer: you haven’t injected anything yet.

**Reply by heyzeuscrist** [reply]
ah thank you

**Comment by R3nvv** [comment]
Mine worked from the first try then it started to ask me for git credentials username and password! I don't know any of them + i tried to look for answers but i couldn't find any. Please help ^^

**Reply by winstxnhdw** [reply]
How are you here with an account right now if you do not know your username and password?

**Comment by Dexteriority** [comment]
It looks complicated for those who used to .exe or autoinjection.
They just wanted a simple yet evil ¯\_(ツ)_/¯

**Reply by winstxnhdw** [reply]
Too bad then because I only distribute executables for non-“cheat” projects. Cheats, mods, hacks, keygens, etc are often associated with malware. This association is further exacerbated by distributing unsigned executables that trigger Windows’ builtin antivirus. It’s also convenient to point directly to the source code they have in their computers if ever accused of shipping malware.

**Reply by Dexteriority** [reply]
Yours are so different man, its been doing a good job since then.
We all kept improving this lc-hax with Suggestion and re-codes.
This is far best "hax" for Lethal Company ever created.[](url)

**Comment by sunRay004** [comment]
There is an error in the command
Command is: "git clone --recursive —depth 50 https://github.com/winstxnhdw/lc-hax.git"
"—depth" should be replaced with "--depth"

**Reply by winstxnhdw** [reply]
Oh, thanks for the heads up. Seems like my phone converts double dashes into a long dash.

**Comment by monkeyboy445236** [comment]
the mod worked great and smoothe but i turned off my computer and when i turned it on it said failed to find injector but it was working fine before i turned it off what do i do

**Reply by winstxnhdw** [reply]
Your anti-virus probably deleted the injector. Just build it again.

**Reply by monkeyboy445236** [reply]
dang you reply fast how do i build it again

**Reply by winstxnhdw** [reply]
Just run `requirements.bat` but your anti-virus will probably delete it again. You can follow the whitelist steps in the README.

**Reply by monkeyboy445236** [reply]
thank you ill go try that

**Reply by monkeyboy445236** [reply]
when i run it it just says "fatal: not a git repository (or any of the parent directories): .git"

**Reply by winstxnhdw** [reply]
That's because you downloaded the code as a zip and extracted it didn't you? Follow the instructions and use `git` to clone the code for you.

**Reply by monkeyboy445236** [reply]
okay sorry if im taking up your time im new to computer stuff

**Reply by winstxnhdw** [reply]
It's alright. First step to computer stuff is reading. My documentation doesn't hide anything from you.

**Reply by monkeyboy445236** [reply]
so im not sure what to do or how to clone

**Reply by monkeyboy445236** [reply]
i have the git comand thing open but when i paste the clone text into it it says "fatal: destination path 'lc-hax' already exists and is not an empty directory."

**Reply by winstxnhdw** [reply]
Yeah, it's exactly as it says. A folder of the same name already exists. You can just delete the folder then clone it again.

**Reply by monkeyboy445236** [reply]
i followed the instrutions on the video and deleted the old one but now its saying "The system cannot find the file ./submodules/SharpMonoInjectorCore/dist/SharpMonoInjector.exe." again

**Reply by winstxnhdw** [reply]
Yeah, looks like your AV is removing it immediately once it is built.

**Reply by monkeyboy445236** [reply]
is it possible to stop that from happening

**Reply by winstxnhdw** [reply]
> You can follow the whitelist steps in the README.

https://github.com/winstxnhdw/lc-hax?tab=readme-ov-file#whitelisting-sharpmonoinjectorcore

There are easier ways to get around it but I don't recommend them.

**Reply by monkeyboy445236** [reply]
thank you so much for your time and patience it worked i appreciate it greatly

**Comment by SunnyHere1** [comment]
It keeps saying that windows cannot find SharpMonoInjector, can anyone help me?
Or is this outdated and im an idiot

**Reply by SunnyHere1** [reply]
i didnt see the post where someone fixed it, ignore this post

**Comment by ServalSoftworks** [comment]
it cant find dotnet?

**Reply by winstxnhdw** [reply]
restart

**Reply by ServalSoftworks** [reply]
> restart

i did but still same error

**Reply by winstxnhdw** [reply]
add the dotnet binary to your PATH

**Reply by ServalSoftworks** [reply]
where would the binary be located

**Reply by winstxnhdw** [reply]
depends where you installed it right

**Reply by ServalSoftworks** [reply]
i did add it to path but i still get it?

**Reply by winstxnhdw** [reply]
send your output for `$env:PATH`

**Reply by ServalSoftworks** [reply]
The filename, directory name, or volume label syntax is incorrect.

**Reply by ServalSoftworks** [reply]
nvm i got it i modified the batch file to manually point to my binary

**Reply by winstxnhdw** [reply]
looks like you’re on command prompt rather than powershell
